sdk/core/allocator/alloc.h

```diff
@@ -301,8 +307,15 @@ MChunkHeader
 
 	/// The unique identifier of the allocator.
 	uint16_t ownerID : OwnerIDWidth;
-	bool     isPrevInUse : 1;
-	bool     isCurrInUse : 1;
+	/**
+	 * Is this a sealed object?  If so, it should be exempted from free in
+	 * `heap_free_all` because deallocation requires consensus between the
+	 * holder of the allocator capability and the holder of the sealing
+	 * capability.
+	 */
```

Comment:
Probably also worth a quick comment that this, like `isCurrInUse`, effectively is a _type tag_ on the body of the chunk, indicating that it is a `struct SObjStruct`.

By that token, it'd be good explicitly zero the bit in `mspace_free` (or maybe `quarantine_pending_push`) and to teach the `ok_free_chunk` consistency test that `isSealedObject` must be `0` (because we can't have both a `struct SObjStruct` and a `MChunk` or `TChunk` in the chunk body).  (At the moment, the bit is ignored through the entire quarantine/free/coalesce part of heap bytes' lifetimes, and will be set or cleared when a header is allocated.)

Reply:
That's probably a good idea when we're debugging.  I'd rather skip the extra instructions, but if we're already clearing the in-use bit then it can be folded into the same read-modify-write instruction.

Reply:
Done.